miri

```diff
@@ -163,8 +163,8 @@ test|test-debug|bless|bless-debug)
     esac
     # Then test, and let caller control flags.
     # Only in root project and ui_test as `cargo-miri` has no tests.
-    $CARGO test $CARGO_BUILD_FLAGS --manifest-path "$MIRIDIR"/Cargo.toml "$@"
     $CARGO test $CARGO_BUILD_FLAGS --manifest-path "$MIRIDIR"/ui_test/Cargo.toml "$@"
+    $CARGO test $CARGO_BUILD_FLAGS --manifest-path "$MIRIDIR"/Cargo.toml "$@"
```

Comment:
Not sure I like that, building the ui_test tests can take a bit and I don't want to wait for that before the main Miri tests run. In fact I often Ctrl-C during the ui_test tests so that I can get my terminal back.^^

Reply:
ah true ^^ it takes milliseconds to run, but the compiling delay is noticeable. Once it's compiled the next time will not be noticeable though unless you touch ui_test, in which case we *do* want to recompile and rerun the tests, so 🤷

Reply:
It's never compiled after a `./rustup-toolchain` though, which is a significant fraction of my builds.